docs/source/protocol_v3.rst

```diff
@@ -1,51 +1,437 @@
-Proposed Protocol v3
-====================
+IMDv3 Protocol
+==============
 
-The suggested changes to the protocol are as follows:
+IMDv3 is a protocol for communicating molecular simulation data through a socket. 
+It allows for two-way communication: via IMDv3, a simulation engine sends data to a receiver 
+and the receiver can send forces and certain requests back to the simulation engine.
```

Comment:
Need to add that it is based on INDv2 with citation.

Reply:
Also talk about history and version numbers. 

- version numbers are monotonously increasing ints
- table with 1 = published version (citation), 2 (actually implemented version in VMD, NAMD, and in the reference implementation (link)), 3 = this specification